app/forms/claims/mentor_form.rb

```diff
@@ -0,0 +1,42 @@
+class Claims::MentorForm < ApplicationForm
+  FORM_PARAMS = %i[trn].freeze
+
+  attr_accessor :school, :trn, :first_name, :last_name
+
+  validate :validate_mentor
```

Comment:
I think the error message is a bit wrong, the message on the prototype appears to be `The mentor has already been added`

![Screenshot from 2024-02-27 09-37-05](https://github.com/DFE-Digital/itt-mentor-services/assets/11318084/cfeed865-12dd-4d69-a792-14703bf1e68c)